examples/chip-tool/templates/logging/DataModelLogger-src.zapt

```diff
@@ -0,0 +1,39 @@
+{{> header}}
+
+#include <commands/clusters/DataModelLogger.h>
+
+{{#zcl_structs}}
+{{#if has_more_than_one_cluster}}
+{{> log_struct_value ns="detail"}}
+{{/if}}
+{{/zcl_structs}}
+
+{{#zcl_clusters}}
+{{#zcl_structs}}
+{{#unless has_more_than_one_cluster}}
+{{> log_struct_value ns=parent.name}}
+{{/unless}}
+{{/zcl_structs}}
+{{/zcl_clusters}}
+
+{{#zcl_clusters}}
+{{#zcl_events}}
+CHIP_ERROR DataModelLogger::LogValue(const char * label, size_t indent, const chip::app::Clusters::{{asUpperCamelCase parent.name}}::Events::{{asUpperCamelCase name}}::DecodableType & value)
+{
+  DataModelLogger::LogString(label, indent, "{");
+{{#zcl_event_fields}}
```

Comment:
Is this going to be generated for embedded platforms as well?

Reply:
Yes. It is already.